modules/account-lib/src/coin/hbar/walletInitializationBuilder.ts

```diff
@@ -1,63 +1,62 @@
 import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
-import { proto } from '../../../resources/hbar/protobuf/hedera';
+import { AccountCreateTransaction, ThresholdKey } from '@hashgraph/sdk';
 import { BuildTransactionError } from '../baseCoin/errors';
 import { TransactionBuilder, DEFAULT_M } from './transactionBuilder';
 import { Transaction } from './transaction';
-import { isValidPublicKey, toUint8Array, toHex } from './utils';
+import { isValidPublicKey, toHex } from './utils';
 import { KeyPair } from './';
 import { TransactionType } from '../baseCoin';
 
 export class WalletInitializationBuilder extends TransactionBuilder {
   private _owners: string[] = [];
-  private _txBodyData: proto.CryptoCreateTransactionBody;
+  private _cryptoBuilder: AccountCreateTransaction;
 
   constructor(_coinConfig: Readonly<CoinConfig>) {
     super(_coinConfig);
-    this._txBodyData = new proto.CryptoCreateTransactionBody();
-    this._txBody.cryptoCreateAccount = this._txBodyData;
-    this._txBodyData.autoRenewPeriod = new proto.Duration({ seconds: 7890000 });
+    this._cryptoBuilder = new AccountCreateTransaction();
   }
 
   // region Base Builder
   /** @inheritdoc */
   protected async buildImplementation(): Promise<Transaction> {
-    this._txBodyData.key = { thresholdKey: this.buildOwnersKeys() };
-    this._txBodyData.initialBalance = 0;
+    this._cryptoBuilder
+      .setInitialBalance(0)
+      .setKey(this.buildOwnersKeys())
+      .setAutoRenewPeriod(7890000);
```

Comment:
hm where is this variable coming from? we could use a constant for this

Reply:
This is a constant that we got from the hedera sdk